smartsim/entity/files.py

```diff
@@ -130,3 +130,122 @@ def _check_path(self, file_path):
         if path.isdir(full_path):
             return full_path
         raise SSConfigError(f"File or Directory {file_path} not found")
```

Comment:
Lets change this to ``FileNotFoundError``. We've abused the ``SSConfigError`` in the past and it's really only supposed to pertain to pre-runtime configuration.

Anywhere that an error is raised about a listed file not found should raise ``FileNotFoundError`` (the builtin error class)

Reply:
Resolved in https://github.com/CrayLabs/SmartSim/pull/119/commits/d92cdfd634d862d2e266fd6d364fd784e37df313